packages/cdktf-cli/lib/models/terraform-cloud.ts

```diff
@@ -147,9 +150,22 @@ export class TerraformCloud implements Terraform {
     }
 
     this.client = new TerraformCloudClient.TerraformCloud(this.token);
+
     this.abortSignal.addEventListener("abort", () => {
       this.removeRun("cancel");
     });
+
+    const httpProxy = process.env.http_proxy || process.env.HTTP_PROXY;
+    if (httpProxy && httpProxy !== "undefined") {
+      // ¯\_(ツ)_/¯
```

Comment:
Prettier moved it to the next line 😅 That comment was for `!== "undefined"` where I couldn't find how this could happen at all..
Should I remove it 😅

Reply:
No, that explains it 👍

Reply:
Ok, I'll leave it as some humor for others to find then 😁